packages/utils/plugins-helpers/src/helpers.ts

```diff
@@ -25,7 +25,7 @@ export function isOutputConfigArray(type: any): type is Types.OutputConfig[] {
 }
 
 export function isConfiguredOutput(type: any): type is Types.ConfiguredOutput {
-  return typeof type === 'object' && type.plugins;
+  return typeof type === 'object' && (type.preset || type.plugins);
```

Comment:
Passing a plugin is currently mandatory; this is a bit overkill when it comes to preset, because presets are then expected to provide their own plugins anyway. In my own case I've built a preset that doesn't require any plugin, since it includes the relevant ones from the get-go. The change is to make it so that having a plugin isn't required if a preset is configured.